recipes/default.rb

```diff
@@ -19,6 +19,11 @@
     action :permissive
 end
 
+package 'cronie'
+service 'crond' do
+    action [:enable, :start]
+end
```

Comment:
It was in the install_ext_services recipe, only when the profile was monolithic or server. It is needed for the accounting cron jobs, but is also needed on websockify role. Also, last week I was setting up in a customer and they added backup scripts in some other servers which ran from cron. I'm happy to move this elsewhere though.